srcts/src/shiny/render.ts

```diff
@@ -79,22 +79,48 @@ async function renderContent(
   }
 }
 
-// This function was introduced in v1.7.5, then deprecated in v1.8.0 once we
```

Comment:
For `renderContent`, please add some information about why this won't work in shinylive (and about the synchronous XHR warnings even outside of shinylive, #789).

And for `renderContentAsync`, tell people that this is what they should use going forward.

Reply:
There's a pretty good (and lengthy) comment just above `renderContentAsync()` on this topic